packages/labs/router/src/routes.ts

```diff
@@ -118,10 +124,12 @@ export class Routes implements ReactiveController {
 
   constructor(
     host: ReactiveControllerHost & HTMLElement,
-    routes: Array<RouteConfig>
+    routes: Array<RouteConfig>,
+    fallback?: BaseRouteConfig
```

Comment:
I think we should put `fallback` in an options bag. If we were to add another param, users would have to pass `undefined` for fallback, which is ungainly.

Reply:
Clarifying - API like so:

```ts
constructor(
    host: ReactiveControllerHost & HTMLElement,
    routes: Array<RouteConfig>,
    options?: {fallback?: BaseRouteConfig}
)
```

Reply:
Yep!

Reply:
Done! Thanks!